### PR TITLE
fix: hb_report: Collect irregular log file

### PR DESCRIPTION
**Pull request by liangxin1300**
Collect irregular time-format log file such as ha-cluster-bootstrap.log.

I will fire a bug if this commit make sense

### CODE REVIEW

**Comment by krig** [comment]
We should definitely collect ha-cluster-bootstrap.log. But I wonder if this might collect too many files...

**Comment by liangxin1300** [comment]
> But I wonder if this might collect too many files...

No, in code, this branch is from config.report.collect_extra_logs or from command-line option, 
so, hb_report will collect what user want it to collect:)

**Comment by krig** [comment]
OK, great :)

File: hb_report/utillib.py [hunk 1]
```diff
@@ -294,6 +294,12 @@ def collect_info():
         if l == constants.HA_LOG and l != constants.HALOG_F:
             os.symlink(constants.HALOG_F, os.path.join(constants.WORKDIR, os.path.basename(l)))
             continue
+        if is_our_log(l, constants.FROM_TIME, constants.TO_TIME) == 4:
+            log_warning("found irregular log file %s" % l)
```
Comment (krig) [inline hb_report/utillib.py]: Should this really be a warning? Maybe info is more appropriate. After all we expect ha-cluster-bootstrap.log to be irregular, for example.

**Comment by krig** [comment]
Merging as it is now, we can change warning to info later if that makes sense. Thanks!

**Pull request merged**